podcasts/Whats New/WhatsNewFullView.swift

```diff
@@ -0,0 +1,91 @@
+import SwiftUI
+import PocketCastsServer
+
+struct WhatsNewFullView: View {
```

Comment:
The goal is for this view to be more generic, but it's a bit specialized on the Slumber announcement due to time constraints. Shall I change the name or keep it?